### PR TITLE
#6808 Fixes anchor modifying YAML File for part six of tutorial

### PR DESCRIPTION
**Pull request by jalmarazg**
This PR is a fix for the issue #6808 .
The problem was a typo in the tutorial-links.yaml file.

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 4d45ea60253f7e46b5f3cded4a11e79d70acac80

https://app.netlify.com/sites/using-postcss-sass/deploys/5b5b8268fdd72a2206fef698

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbygram* failed.

Built with commit 4d45ea60253f7e46b5f3cded4a11e79d70acac80

https://app.netlify.com/sites/gatsbygram/deploys/5b5b8267fdd72a2206fef692

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 4d45ea60253f7e46b5f3cded4a11e79d70acac80

https://app.netlify.com/sites/gatsbyjs/deploys/5b5b8267fdd72a2206fef690

File: www/src/data/sidebars/tutorial-links.yaml [hunk 1]
```diff
@@ -71,7 +71,7 @@
         - title: What are transformer plugins?
           link: /tutorial/part-six/#transformer-plugins
         - title: Create a list of our site's markdown files
-          link: /tutorial/part-six/#create-a-list-of-our-sites-markdown-files-in-srcpagesindexjs
+          link: /tutorial/part-six/#create-a-list-of-your-sites-markdown-files-in-srcpagesindexjs
```
Comment (piotrkwiecinski) [inline www/src/data/sidebars/tutorial-links.yaml]: The hash should be `create-a-list-of-your-sites-markdown-files-in-code-classlanguage-textsrcpagesindexjscode`

Here is the html generated on that header element:
```
<a href="#create-a-list-of-your-sites-markdown-files-in-code-classlanguage-textsrcpagesindexjscode" aria-hidden="true" class="anchor"></a>
```

Comment (KyleAMathews) [reply www/src/data/sidebars/tutorial-links.yaml]: hrmmm we need to fix whatever is causing the language codes to be added to the hashes.

Comment (jalmarazg) [reply www/src/data/sidebars/tutorial-links.yaml]: I just made a commit with ```create-a-list-of-your-sites-markdown-files-in-code-classlanguage-textsrcpagesindexjscode``` instead the first change I initially made.

Comment (piotrkwiecinski) [reply www/src/data/sidebars/tutorial-links.yaml]: @KyleAMathews langeuage codes are added due to backticks in h2 for filename.

Shall we investigate it as separate issue?

Comment (pieh) [reply www/src/data/sidebars/tutorial-links.yaml]: PR that solves backticks problem in header ids - https://github.com/gatsbyjs/gatsby/pull/6856

**Comment by pieh** [comment]
Language codes in header ids were fixed by #6856, so I adjusted link

**Pull request merged**